bundles/pixi.js-legacy/test/index.js

```diff
@@ -1,8 +1,23 @@
 /* eslint-disable global-require */
-const PIXI = require('../');
-
 describe('PIXI', function ()
 {
+    before(function (done)
+    {
+        this.head = document.querySelector('head');
+        this.script = document.createElement('script');
+        this.script.onload = () => done();
+        this.script.src = require.resolve('../dist/pixi-legacy');
+        this.head.appendChild(this.script);
```

Comment:
What is the benefit of putting these on `this` rather than just having them be vars in the describe scope?

Reply:
No benefit, mostly best practice. In general I like to be able to setup and tear down in mochas conventions

Reply:
Sorry I didn't meant to not set them here, just that the vars would be scoped to the describe:

```js
describe(function ()
{
    let head, script;

    before(function () { head = ... });
    after(function () { ... });
});
```

Doesn't matter, just curious.